probe-rs/src/architecture/arm/ap/mod.rs

```diff
@@ -123,7 +124,24 @@ impl<T: DapAccess> ApAccess for T {
         log::debug!("Reading register {}", R::NAME);
         let raw_value = self.read_raw_ap_register(port.into().ap_address(), R::ADDRESS)?;
 
-        log::debug!("Read register    {}, value=0x{:x?}", R::NAME, raw_value);
+        let is_drw = R::NAME == "DRW";
+        let drw_map = if is_drw {
```

Comment:
This could be wrapped with `if log::log_enabled!(log::Level::Debug) {`, then it's only done when logging is active.